test-consumers/compose/shared/src/jsMain/kotlin/uk/co/baconi/session/getRedirectUri.js.kt

```diff
@@ -0,0 +1,5 @@
+package uk.co.baconi.session
+
+import io.ktor.http.*
+
+actual fun getRedirectUri(): Url = URLBuilder("http://localhost:8180/callback").build()
```

Comment:
Replace with probably `http://localhost:8081` where the JS app runs, which can then detect `code` or `error` parameters.